.gitignore

```diff
@@ -1,5 +1,6 @@
 /build*
 */build
+Dockerfiles/Makefile
```

Comment:
This is just because I have a local `Makefile` to automatically set up the image, namespace, and tag, and so I don't need to remember the syntax for the build and run commands. I can include it if it would be useful.